Web/Admin.UI/src/app/components/core/link-admin-subnav-bar/link-admin-subnav-bar.component.html

```
@@ -0,0 +1,27 @@
+    <li class="subnav-item" *ngFor="let item of subnavItems" [class.dropdown]="item.children">
+      <!-- Regular nav item -->
+      <a *ngIf="!item.children" [routerLink]="[item.path!]" routerLinkActive="active">
+        {{ item.label }}
+      </a>
+
+      <!-- Dropdown menu -->
+      <ng-container *ngIf="item.children">
+        <a href="#">
+          {{ item.label }}
+          <div class="dropdown-icon">
+            <vd-icon filePath="../../../assets/icons/chevron-default.svg"></vd-icon>
```

HTMLHint (1.5.0)

[error] The attribute name of [ filePath ] must be in lowercase.

(attr-lowercase)